reconcile/utils/terrascript_client.py

```diff
@@ -3155,6 +3175,154 @@ def is_elasticsearch_domain_name_valid(name):
         pattern = r'^[a-z][a-z0-9-]+$'
         return re.search(pattern, name)
 
+    @staticmethod
+    def elasticsearch_log_group_identifier(
+        domain_identifier: str,
+        log_type: ElasticSearchLogGroupType
+    ) -> str:
+        log_type_name = log_type.value.lower()
+        return f'OpenSearchService__{domain_identifier}__{log_type_name}'
+
+    def _elasticsearch_aggregate_log_groups_per_account(
```

Comment:
nit: from the method name i would have expected some aggregated return per account or filtered ElasticSearchLogGroupInfo objects for a specific account.

Reply:
You are absolutely right! The method name does not make any sense.

Renamed it to `_elasticsearch_filter_all_log_group_infos`.